src/App/ElementMap.cpp

```diff
@@ -542,6 +542,9 @@ void ElementMap::addPostfix(const QByteArray& postfix, std::map<QByteArray, int>
 MappedName ElementMap::setElementName(const IndexedName& element, const MappedName& name,
                                       long masterTag, const ElementIDRefs* sid, bool overwrite)
 {
+    if (element.isNull()) {
```

Comment:
`element.isNull()` is the same as `!element`. So, this change converts the block below into dead code.

Reply:
Sorry, I misunderstood where the error trap should be placed, I'll submit a new PR with a single commit as I'm using the web interface today.